packages/components/src/styles/Heading.module.css

```diff
@@ -1,2 +1,7 @@
 .heading {
+	white-space: nowrap;
+	max-width: calc(100% - var(--lp-size-32));
```

Comment:
There's a small downside to this `max-width`: what if the modal is _not_ dismissible and doesn't have the button? We're not going to make use of the 32px, which seems small, but could contain literally 2-3 letters. And with this rule they'd be truncated. It's a bit of an edge case for sure, but still.

I was playing around with the layout in DevTools and it seemed a bit strange to me that we rely on `flex-wrap: wrap` to ensure the subtitle appears below the heading. Seems like using a grid could help simplify things a bit and avoid that `max-width`.

We can rethink that after this quickfix though. This at least will deal with super long titles better.